src/components/Card/Card.tsx

```diff
@@ -42,8 +36,13 @@ const Card: React.FC<CardProps> = ({ resource }) => {
         variant="classic"
         asChild
       >
-        <Link href={homepage && homepage[0].id ? homepage[0].id : ""}>
-          <Inset clip="padding-box" side="top">
+        <CanopyLink href={homepage && homepage[0].id ? homepage[0].id : ""}>
```

Comment:
This swap fixes the issue temporarily resolved in the earlier PR
https://github.com/canopy-iiif/canopy-iiif/pull/198/files making it so we no long need this:

https://github.com/canopy-iiif/canopy-iiif/blob/5988bdf049686d86ba74744738cd3037ce99a711/src/components/MDX/Card.tsx#L14